ROADMAP.md

```diff
@@ -0,0 +1,94 @@
+# io.js Roadmap
+
+***This is a living document, it describes our policy and priorities as they exist today but can evolve over time.***
+
+## Stability Policy
+
+The single more important consideration in every code change is the impact it will have, positive or negative, on the ecosystem (modules and applications).
+
+io.js does not remove JavaScript API.
```

Comment:
@dshaw, fortunately that is really unlikely to happen again due to how stable releases of io.js will also track stable versions of V8, which is a guarantee io.js will never "unship" shipped V8 features.